src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -147,70 +140,64 @@ public void testAntlrError(@SysErr Capturable systemErr) throws Exception {
+                "InputAbstractJavadocParsingErrors2.java"), expectedMessagesForFile1);
+
+        final String[] expectedMessagesForFile2 = {
+            "9: " + getCheckMessage(MSG_JAVADOC_PARSE_RULE_ERROR, 78,
+                    "mismatched input '(' expecting <EOF>", "JAVADOC"),
+        };
+        verifyWithInlineConfigParser(getPath(
+                "InputAbstractJavadocInvalidAtSeeReference2.java"), expectedMessagesForFile2);
+
         assertWithMessage("Error is unexpected")
-            .that(systemErr.getCapturedData())
-            .isEqualTo("");
+                .that(systemErr.getCapturedData())
+                .isEqualTo("");
     }
 
     @Test
     public void testCheckReuseAfterParseErrorWithFollowingAntlrErrorInSingleFile()
             throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(TempCheck.class);
         final String[] expected = {
-            "8: " + getCheckMessage(MSG_JAVADOC_MISSED_HTML_CLOSE, 4, "unclosedTag"),
-            "13: " + getCheckMessage(MSG_JAVADOC_PARSE_RULE_ERROR, 82,
+            "9: " + getCheckMessage(MSG_JAVADOC_MISSED_HTML_CLOSE, 4, "unclosedTag"),
+            "16: " + getCheckMessage(MSG_JAVADOC_PARSE_RULE_ERROR, 82,
                     "mismatched input '(' expecting <EOF>", "JAVADOC"),
         };
-        verify(checkConfig,
+        verifyWithInlineConfigParser(
             getPath("InputAbstractJavadocUnclosedTagAndInvalidAtSeeReference.java"), expected);
     }
 
     @Test
-    public void testPosition()
-            throws Exception {
+    public void testPosition() throws Exception {
         JavadocCatchCheck.clearCounter();
-        final DefaultConfiguration checkConfig = createModuleConfig(JavadocCatchCheck.class);
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputAbstractJavadocPosition.java"), expected);
+        verifyWithInlineConfigParser(getPath("InputAbstractJavadocPosition.java"), expected);
         assertWithMessage("Invalid number of javadocs")
             .that(JavadocCatchCheck.javadocsNumber)
-            .isEqualTo(65);
+            .isEqualTo(130);
     }
 
     @Test
-    public void testPositionWithSinglelineComments()
-            throws Exception {
+    public void testPositionWithSinglelineComments() throws Exception {
         JavadocCatchCheck.clearCounter();
-        final DefaultConfiguration checkConfig = createModuleConfig(JavadocCatchCheck.class);
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig,
-            getPath("InputAbstractJavadocPositionWithSinglelineComments.java"), expected);
+        verifyWithInlineConfigParser(
+                getPath("InputAbstractJavadocPositionWithSinglelineComments.java"), expected);
         assertWithMessage("Invalid number of javadocs")
             .that(JavadocCatchCheck.javadocsNumber)
-            .isEqualTo(65);
+            .isEqualTo(130);
```

Comment:
This is strange.
Please recheck how it might be related to `JavadocCatchCheck.clearCounter()` .
It sounds like it use to be single instance for whole test class execution, but as we move to be created in each config, it becomes different.
It might be a reason of lost coverage